### DISCUSSION TITLE
PSA: Windows 10 support

### DISCUSSION THREAD

**Discussion by valinet** [valinet/ExplorerPatcher#898] [Announcements]
Hi

I have taken the time and fixed the few things that made ExplorerPatcher not able to run on Windows 10. Thus, the program is now able to run on Windows 10, of course, without the Windows 11-specific features (like choosing the taskbar style, for e.g. etc). Generic features are all available, including the weather widget and Simple Window Switcher.

I have tested it on Windows 10 LTSC 2021 (Version 22H2 - 19044.1526) and Windows 10 LTSC 2019 (Version 1809 - 17763.2565). It *may* work on older versions, but I honestly don't think so. Needless to say it is totally unsupported on ancient Windows 10 versions, and unlikely to ever get fixed. Likewise for older Windows versions.

Also, please refrain from importing saved settings from ExplorerPatcher on Windows 11 to the Windows 10 version; this scenario is not supported at the moment and may lead to weird behaviors.

![image](https://user-images.githubusercontent.com/6503598/155621632-70c40bdc-38cc-4180-9923-08e262544a53.png)

Thanks.

**Comment by Anixx** [comment]
Great! Is the Classic theme supported?

**Reply by valinet** [reply]
Probably. Try out and see, I haven't tested that. But it should work the same as in Windows 11 if using a similar method of configuring it.
